.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    ["es2015", { "modules": false }]
+  ]
```

Comment:
It is necessary for tree shaking, webpack 2 can handle ES6 modules syntax. Tree shaking eliminates unused exports.